.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+Hello, and welcome! Thanks you for your contribution to **Gulliver**.
+
+1. Please let us know about the contribution you are making
+   - [ ] Does this contribution update documentation?
+   - [ ] Does this contribution update code?
+      - [ ] Does it fix a bug?
+      - [ ] Does it add new functionality?      
+
+1. Assuming there is a code change in this contribution
+   - [ ] Have you included unit test to verify the change?
+   - [ ] Do all pre-existing unit tests pass as expected?
+   - [ ] Do the change break existing functionality, if so have you explained why is such appropriate?
```

Comment:
Does the change break existing functionality, if so have you explained why doing so is necessary?